VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: Spacemouse Gr_v_ECU 


BO_ 387 ControlMessage: 6 Spacemouse
 SG_ Rotate_RL : 40|8@1- (1,0) [0|0] "" Gr_v_ECU
 SG_ Tilt_RL : 32|8@1- (1,0) [0|0] "" Gr_v_ECU
 SG_ Tilt_FR : 24|8@1- (1,0) [0|0] "" Gr_v_ECU
 SG_ Parallell_FR : 8|8@1- (1,0) [0|0] "" Gr_v_ECU
 SG_ Parallelll_RL : 0|8@1- (1,0) [0|0] "" Gr_v_ECU
 SG_ UpDown : 16|8@1- (1,0) [0|0] "" Gr_v_ECU

BA_DEF_  "DBName" STRING ;
BA_DEF_  "BusType" STRING ;
BA_DEF_ BO_  "VFrameFormat" ENUM  "StandardCAN","ExtendedCAN","reserved","J1939PG";
BA_DEF_DEF_  "DBName" "can_signals";
BA_DEF_DEF_  "BusType" "CAN";
BA_ "VFrameFormat" BO_ 387 0;
